### DISCUSSION TITLE
Nachrichten überschneiden sich

### DISCUSSION THREAD

**Discussion by darkbandit82** [pixelit-project/PixelIt#281] [Q&A]
Wie habt ihr das Problem gelöst. dass sich Nachrichten nicht überschneiden? Oder gibt es da keine Lösung momentan?

**Comment by foorschtbar** [comment]
Erzähl mal bissl was zu deinem Setup. Ich habe bereit eine Vermutung warum das bei dir nicht klappt.

**Comment by darkbandit82** [comment]
Hi danke das Du dich meldest. So sieht es momentan bei mir aus. Es ist so das wenn zwei Nachrichten zur gleichen Zeit kommen. dass eine Nachricht nur kurz angezeigt wird.

![pv](https://github.com/pixelit-project/PixelIt/assets/132387267/b5415e5f-9294-4307-b4ee-c89b748e2e4c)

**Reply by foorschtbar** [reply]
Da fehlt das Konzept der Playlists. Schau dir das mal bitte in der Doku an. Die Daten gehen oben an den Screen Data Update und die Playlist kümmert sich drum die nacheinander anzuzeigen für die definierte Zeit.

**Reply by darkbandit82** [reply]
Danke dir Ich werde es mal probieren
